includes/admin/class-wc-stripe-inbox-notes.php

```diff
@@ -31,6 +32,10 @@ public function __construct() {
 		}
 	}
 
+	public static function create_upe_availability_note() {
+		WC_Stripe_UPE_Availability_Note::init();
```

Comment:
The compatibility issue would still persist, because on `WC_VERSION` with value < than `4.4.0` the `WC_Stripe_UPE_Availability_Note` class would not exist (because its file is not included). So this line would cause a fatal error.
```suggestion
        if ( version_compare( WC_VERSION, '4.4.0', '>=' ) ) {
            require_once WC_STRIPE_PLUGIN_PATH . '/includes/notes/class-wc-stripe-upe-availability-note.php';
    		WC_Stripe_UPE_Availability_Note::init();
		}
```